core/src/env_platform/db_handler.bal

```diff
@@ -74,12 +80,15 @@ function deleteApplication(string applicationId) returns boolean|error {
     if (applicationStatus == "draft") {
         int|error deleted = applicationCollection->delete({"applicationId": applicationId, "status": "draft"});
         if (deleted is int) {
+            log:printDebug("Deleted application count: " + deleted.toString());
             return deleted == 1 ? true : false;
         } else {
             // Returns the error.
+            log:printDebug("An error occurred while deleting the draft with the application ID: " + applicationId + ". " + deleted.reason() + ".");
             return deleted;
         }
     } else {
+        log:printDebug("Cannot delete the application with application ID: " + applicationId + " as it is already submitted.");
```

Comment:
```suggestion
        log:printDebug("Cannot delete the application with application ID: " + applicationId + " since it is already submitted.");
```